HW04.Rmd

```diff
@@ -1,126 +1,306 @@
-* Fall = October-December
-
-```{r season}
-
-```
-
-
-# These are more open ended questions from here on out, you could look at making multiple summaries or types of plots. 
-
-### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
-
-```{r race}
-
-```
-
-(hint maybe looking at percentages would be good)
-```{r race_2}
-
-```
-
-
-### Are police-involved gun deaths significantly different from other gun deaths? Assess the relationship between police involvement and age, police involvement and race, and the intersection of all three variables.
-
-```{r police}
-
-```
-
-Think about performing stats such as a t.test to see if differences you are plotting are statistically significant
-
-```{r stats?}
-
-
-```
-
-
-```{r police_2}
-
-```
-
-```{r stats_2?}
-
-
-```
-
-
-Finally, all together now:
-```{r police_3}
-
-```
-
-
-## Session info
-
-```{r}
-# always good to have this for reproducibility purposes
-devtools::session_info()
-```
-
+---
+title: "Exploring Gun Deaths in America"
+author: Ariana Tortolani
+date: 07/27/2020
+output: github_document
+---
+
+## Objectives
+* To explore gun death data curated by FiveThirtyEight [original data available here](https://github.com/fivethirtyeight/guns-data) by
+    * Month
+    * Intent
+    * Age and sex
+    * Education
+    * Season
+    * Intent by race
+    * Police-relatedness 
+
+## Read in the data
+
+The gun_deaths.csv is provided in the repository. 
+```{r load-data, message=FALSE, warning=FALSE}
+library(tidyverse)    # load tidyverse packages, including ggplot2
+library(ggplot2)
+library(knitr)
+
+# read in the data
+gun_deaths <- read.csv("gun_deaths.csv")
+
+#initial look at data
+glimpse(gun_deaths)
+
+#change month col (numbers) to month names
+gun_deaths$month <- factor(gun_deaths$month, labels = c("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"))
+
+```
+
+## Generating a data frame that summarizes the number of gun deaths per month and printing using `kable()`. 
+
+Knitr is part of the knitr package. Allows you to change column names and add captions and make pretty tables in your knitted document. Hint: set the argument format = "markdown"
+
+
+```{r month, warning=FALSE}
+
+deaths_per_month <- gun_deaths %>%
+    group_by(month) %>%
+    count()
+
+kable(deaths_per_month, format = "markdown",
+      col.names = c("Month", "Number of Deaths"),
+      caption = "Number of gun related deaths per month.")
+
+```
+
+
+### Generating a bar chart with human-readable labels on the x-axis. That is, each month should be labeled "Jan", "Feb", "Mar" (full or abbreviated month names are fine), not `1`, `2`, `3`.
+
+```{r month_chart}
+#create plot
+ggplot(gun_deaths, aes(month)) +
+    geom_bar(fill = "cornflowerblue") +
+    scale_y_continuous(breaks = seq(0, 9000, by = 1000)) +
+    
+#add labels
+    labs(title = "Gun Deaths per Month from 2012-2014",
```

Comment:
Nice figure!
"Total Gun Deaths by Month from 2012-2014" may be a more accurate title, but that's a bit pedantic.